homeassistant/components/gios/__init__.py

```diff
@@ -52,12 +52,12 @@ async def _async_update_data(self):
         """Update data via library."""
         try:
             with timeout(30):
-                await self.gios.update()
+                data = await self.gios.async_update()
```

Comment:
Why not return right here instead of storing the data remporarily?

```suggestion
                return await self.gios.async_update()
```

Reply:
Of course you're right, this has no sense 🤦‍♂️